layers/meta-raspberrypi

```diff
@@ -1 +1 @@
-Subproject commit dedd75f08ffe3360722ab761da18987aa681edd1
+Subproject commit 953dc5a406be6e1ad78c8aaf90df66ff4d7e5275
```

Comment:
Apparently the jethro branch is not receiving backports/additions so guess this will do for now